Documentation/templates/strings/flat-swift4.md

```diff
@@ -21,6 +21,7 @@ You can customize some elements of this template by overriding the following par
 | `enumName` | `L10n` | Allows you to change the name of the generated `enum` containing all string tables. |
 | `noComments` | N/A | Setting this parameter will disable the comments describing the translation of a key. |
 | `publicAccess` | N/A | If set, the generated constants will be marked as `public`. Otherwise, they'll be declared `internal`. |
+| `runtime` | N/A | If set to true, the generated strings will be computed instead of static. |
```

Comment:
The term "static" here is probably not the most appropriate 😉 they are still `static` in the sense `static var` (properties of the _type_ not the _instance_), they are just recomputed at each call instead of constants lazily computed only once.